### DISCUSSION TITLE
Add click to play button to play the content

### DISCUSSION THREAD

**Discussion by sabersalv** [jesec/flood#393] [Ideas]
see https://github.com/sabersalv/flood/pull/11, how do your guys think of this?

**Comment by jesec** [comment]
I like the idea, but the URL scheme is likely not portable across different players or platforms.

**Reply by sabersalv** [reply]
macOS works fine, I didn't test other platforms, mobile does not support this, which can be hidden, and I don't know about Windows. And can have a setting let user choose which player he want to use.

**Reply by jesec** [reply]
It would only work with IINA. Nothing else. 

I don't think it would be possible to have the options for all custom URL schemes for all sort of players users may use. It would be too bloated.
